website/content/blog/2024-12-01-a-year-in-review.md

```diff
@@ -0,0 +1,114 @@
+  * [Detect GamePad/Keyboard events on Android](https://github.com/MonoGame/MonoGame/pull/8465)
+  * [Remove string allocations when calling GamePad.GetCapabilities](https://github.com/MonoGame/MonoGame/pull/8453)
+  * [Content Pipeline now works on M1/M2 Macs](https://github.com/MonoGame/MonoGame/pull/8570)
+  * [Fix Android MediaPlayer Song implementation](https://github.com/MonoGame/MonoGame/pull/8583)
+  * [Update MonoGame.Library.OpenAL to 1.23.1.10](https://github.com/MonoGame/MonoGame/pull/8560)
+  * [Fix Sprite font multi byte parsing](https://github.com/MonoGame/MonoGame/pull/8554)
+  * [Added Vibration and Caps to Native GamePad API](https://github.com/MonoGame/MonoGame/pull/8520)
+  * [Fix MacOS Content directory location](https://github.com/MonoGame/MonoGame/pull/8479)
+  * [Detect GamePad/Keyboard events on Android](https://github.com/MonoGame/MonoGame/pull/8465)
+
+* #### Game Released (that we know of)
+  * [Breeze & Freeze](https://msiebenmann.itch.io/breeze-and-freeze) by msiebenmann
+  * [Tic Tac Dough](https://ms00.itch.io/tic-tac-dough) by MS00
+  * [Ninja Cat Remewstered](https://pixelshock.itch.io/ninja-cat-remewstered) by Pixel Shock
+  * [Smack 'n' Snatch](https://skyyyla.itch.io/smack-n-snatch) by Skyla
+  * [FishGrid](https://shmellyorc.itch.io/fishgrid) by Shmellyorc
+  * [Axe Attacks](https://serork.itch.io/axe-attacks) by Serork
+  * [Mochi's Escape](https://anonames-lair.itch.io/mochi-escape) by Anonames
+  * [Armored Crucible: The Crimson Dunes](https://fontty.itch.io/ac-tcd) by Fontty Games
+  * [Arid Arnold](https://icefish-software.itch.io/arid-arnold) by IceFish Software
+  * [Dungeon Adventure Gang](https://store.steampowered.com/app/2147690/Dungeon_Adventure_Gang/) by DAG Team
+  * [Cranky Chibi Cthulhu](https://little-tlaloc-interactive.itch.io/cranky-chibi-cthulhu) by Little Tlaloc Interactive. Uses [FlatRedBall](https://flatredball.com/) and [Gum](http://docs.flatredball.com/gum/)
+  * [BattleCrypt Bombers](https://store.steampowered.com/app/2188930/BattleCrypt_Bombers/) by Narfox LLC. Uses [FlatRedBall](https://flatredball.com/) and [Gum](http://docs.flatredball.com/gum/)
+
+
+### What's coming in the next year:
+
+* #### 3.8.3 Release
+
+  * Smoother macOS installation support and Content Pipeline  
+  * Bump to use latest StbImageSharp  
+  * more????  
+* The MonoGame.Framework.Native is going to be a principal focus. The plan is to allow the support of NativeAOT on all the platforms which support it.   
+* Upgrading the Shader Compiler to support more modern shader models.   
+* Improved Content pipeline (Mr H / Harry, details)  
+  * Automating XNB builds to make using MGCB or the content editor optional, and adding content more easily to projects  
+* Full DX12 support on Windows / Xbox  
+* Vulkan support to replace OpenGL (Tom to clarify)
```

Comment:
I would say:

- New Vulkan Desktop platform (it may replace OpenGL at some point).